booleans.js

```diff
@@ -0,0 +1,124 @@
+function negate(a) {
+  return !a;
+};
+
+function both(a, b) {
+  if (a == true && b == true) {
+    return true;
+  } else {
+    return false;
+  };
+
+};
+
+function either(a, b) {
+  if (a == true | b == true) {
+
+    return true;
+  } else {
+
+    return false;
+  };
```

Comment:
As above, you could shorten this quite a lot by just returning the result of the initial evaluation:
```
  return a || b;
```
Look carefully at the error eslint is giving you too. What youre actually doing here is using a bitwise operator and not the 'OR' operator, which is `||` and not just `|`